src/Microsoft.Health.Dicom.Api/Controllers/WorkitemController.Retrieve.cs

```diff
@@ -18,14 +25,30 @@ public partial class WorkitemController
     /// This action requests a UPS Instance on the Origin-Server. It corresponds to the UPS DIMSE N-GET operation.
     /// </summary>
     [HttpGet]
+    [AcceptContentFilter(new[] { KnownContentTypes.ApplicationDicomJson })]
+    [Produces(KnownContentTypes.ApplicationDicomJson)]
+    [ProducesResponseType(typeof(DicomDataset), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType((int)HttpStatusCode.NotFound)]
+    [ProducesResponseType((int)HttpStatusCode.Conflict)]
     [VersionedPartitionRoute(KnownRoutes.RetrieveWorkitemInstancesRoute, Name = KnownRouteNames.VersionedPartitionRetrieveWorkitemInstance)]
     [PartitionRoute(KnownRoutes.RetrieveWorkitemInstancesRoute, Name = KnownRouteNames.PartitionedRetrieveWorkitemInstance)]
     [VersionedRoute(KnownRoutes.RetrieveWorkitemInstancesRoute, Name = KnownRouteNames.VersionedRetrieveWorkitemInstance)]
     [Route(KnownRoutes.RetrieveWorkitemInstancesRoute, Name = KnownRouteNames.RetrieveWorkitemInstance)]
     [AuditEventType(AuditEventSubType.RetrieveWorkitem)]
-    public IActionResult RetrieveAsync()
+    public async Task<IActionResult> RetrieveAsync(string workitemInstanceUid)
     {
-        _logger.LogInformation("Requesting non-implemented endpoint.");
-        return new StatusCodeResult((int)HttpStatusCode.NotFound);
+        _logger.LogInformation("Search workitem.");
```

Comment:
We don't need these kinds of log statements - there are logs being generated at the framework level that capture action method calls.